src/core/unicode.js

```diff
@@ -1640,10 +1640,32 @@ function reverseIfRtl(chars) {
   return buf.join("");
 }
 
+function isWhitespace(c) {
+  return (
+    c === " " ||
+    c === "\n" ||
+    c === "\t" ||
+    c === "\r" ||
+    c === "\f" ||
+    c === "\v" ||
+    c === "\u00a0" ||
+    c === "\u1680" ||
+    c === "\u2000" ||
+    c === "\u200a" ||
```

Comment:
Isn't there a bunch of characters missing here, since https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet#character_classes contains the following information:
`[ \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]`

Note in particular the hyphen between `\u2000` and `\u200a`, which indicates a range of characters.